### DISCUSSION TITLE
flet build windows or apk showing a blank page affter successful build, and its makeshift solution for windows.

### DISCUSSION THREAD

**Discussion by jsyoo3722** [flet-dev/flet#3371] [Q&A]
flet 0.19 is out, and I used it like I was told by one of the contributors here.
This version shows why the page failed to load.

My issue gave me this error message:

> Traceback (most recent call last):
>   File "<string>", line 40, in \<module\>
>   File "<frozen runpy>", line 229, in run_module
>   File "<frozen runpy>", line 88, in _run_code
>   File "C:\Users\jjj\AppData\Local\Temp\serious_python_tempca5826b\main.py", line 2, in \<module\>
> ModuleNotFoundError: No module named 'selenium'

It seems the necessary modules are not being loaded? or copied? to serious_python during the build process.

For windows apps, I solved it by manually copying the related folders to my app's site-packages folder.
> COPY the modules being imported
> FROM .\project_folder\\.venv\Lib\site-packages
> TO .\proejct_folder\build\windows\Lib\site-packages

It kept popping up different ModuleNotFoundError messages, so I kept copying all the necessary folders to my app's site-packages folder and voilà~ it works now.

However, for android apks, I tried unzziping the apk, but I couldn't figure out where I should copy my imported modules to...
I hope there will be some viable solutions soon.

**Comment by taaaf11** [comment]
Give your `requirements.txt`
